fillArray.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Creates a new array containing the elements of an existing array repeated
+ * @since 4.0.0
+ * @category Array
+ * @param {Array} array The array to repeat
+ * @param {Number} times An integer specifying how many times elements of the existing array should be repeated
+ * @returns {Array} an array containing the elements of the existing array repeated
+ * @example fillArray([1, 2, 3], 3) // returns [1, 2, 3, 1, 2, 3, 1, 2, 3]
+ */
+function fillArray(array, times) {
```

Comment:
We might need to validate the type for the inputs. Should we add tests for invalid inputs?

Reply:
Also `if` we are renaming this to `arrayRepeat`, we might need to update the local variable names also